installer/win/SymphonyWinInstallerValidatorAction/SymphonyWinInstallerValidatorAction/CustomAction.cs

```diff
@@ -0,0 +1,102 @@
+using System;
```

Comment:
will this only require .net3.5 or higher.  most customer will have upgraded to newer version of .net but out of the box win7 only has .net3.5.

Reply:
you are right, this isn't working on Win7. I'll need to find a soln. for it.